client/app/scripts/superdesk-publish/publish.js

```diff
@@ -18,6 +19,32 @@
         };
 
         var service = {
+            fetchSubscribers: function(criteria) {
+                criteria = criteria || {};
+                criteria.max_results = 200;
```

Comment:
I removed max_results (it was leftover code from previous work). Pagination (or filtering) will be added for subscribers, output channels and destinations in another PR.